common/src/Model/GameConfig.h

```diff
@@ -88,8 +88,9 @@ namespace TrenchBroom {
             IO::Path palette;
             std::string attribute;
             IO::Path shaderSearchPath;
+            std::vector<std::string> excludes;
```

Comment:
Needs a comment explaining that the strings are glob patterns that apply to texture names.